.github/workflows/ci.yml

```diff
@@ -270,6 +270,39 @@ jobs:
 
           _make PLATFORM=virt
 
+  QEMUv7_check:
+    name: make check (QEMUv7)
+    runs-on: ubuntu-latest
+    container: jforissier/optee_os_ci:qemuv8_check2
+    steps:
+      - name: Remove /__t/*
+        run: rm -rf /__t/*
+      - name: Restore build cache
+        uses: actions/cache@v3
+        with:
+          path: /github/home/.cache/ccache
+          key: qemuv7_check-cache-${{ github.sha }}
+          restore-keys: |
+            qemuv7_check-cache-
+      - name: Checkout
+        uses: actions/checkout@v4
+      - shell: bash
+        run: |
+          # make check task
+          set -e -v
+          export LC_ALL=C
+          export BR2_CCACHE_DIR=/github/home/.cache/ccache
+          export CFG_TEE_CORE_LOG_LEVEL=0
+          WD=$(pwd)
+          cd ..
+          TOP=$(pwd)/optee_repo_qemu_v8
+          /root/get_optee_qemuv8.sh ${TOP}
+          mv ${TOP}/optee_os ${TOP}/optee_os_old
+          ln -s ${WD} ${TOP}/optee_os
+          cd ${TOP}/build
+
+          make -j$(nproc) check CFG_LOCKDEP=y CFG_LOCKDEP_RECORD_STACK=n CFG_ENABLE_EMBEDDED_TESTS=n CFG_IN_TREE_EARLY_TAS=pkcs11/fd02c9da-306c-48c7-a49c-bbd827ae86ee CFG_PKCS11_TA=y
```

Comment:
why disabling `CFG_ENABLE_EMBEDDED_TESTS`? (maybe inherited from the IBART setup)

Reply:
Yes. I will remove this flag.

Reply:
Removed